### ISSUE TITLE
Problem with mask and background

### ISSUE THREAD

**Issue by vahidEtt** [suoxinkey/PlenOctrees_NeRF-SH#1]
Hi,

I tried to train with sport_1 dataset, with below config, but it seems "USE_MASK" option doesn't work and I still see the background. Any idea what could be the problem?

SOLVER:
  OPTIMIZER_NAME: "Adam"
  BASE_LR: 0.0005
  WEIGHT_DECAY: 0.0000000
  IMS_PER_BATCH: 2
  START_ITERS: 3000
  END_ITERS: 60000
  LR_SCALE: 0.1
  WARMUP_ITERS: 0

  MAX_EPOCHS: 100
  CHECKPOINT_PERIOD: 3000
  LOG_PERIOD: 30
  BUNCH: 1000
  COARSE_STAGE: 0

INPUT:
  SIZE_TRAIN: [256,192]
  SIZE_TEST: [256,192]

DATASETS:
  TRAIN: "/home/Downloads/sport_1"
  NUM_FRAME: 1
  SHIFT: 0.0
  MAXRATION: 0.0
  ROTATION: 0.0
  USE_MASK: True

DATALOADER:
  NUM_WORKERS: 8

MODEL:
  COARSE_RAY_SAMPLING: 64
  FINE_RAY_SAMPLING: 128
  SAMPLE_METHOD: "NEAR_FAR"       #  "NEAR_FAR"     "BBOX" 
  BOARDER_WEIGHT: 1e10
  SAME_SPACENET: True
  TKERNEL_INC_RAW: True
  USE_SH: True
  DEVICE_IDS: [0]

TEST:
  IMS_PER_BATCH: 1

OUTPUT_DIR: "/media/Data_DL_HD/projects/PlenOctrees_NeRF-SH/output/sport_1"

![Screenshot-20210416220214-1149x880](https://user-images.githubusercontent.com/75507961/115098699-6eeff200-9eff-11eb-8d73-af310c7e38f0.png)

**Comment by suoxinkey** [comment]
Because the mask provided by the datasets is not so perfect. And We only sample the rays inside the mask during training and there is no supervision in the background area. so the above situation will occur.
